pkg/format/format.go

```diff
@@ -171,7 +172,14 @@ func format(d, label, fsType string, forced bool) error {
 		return fmt.Errorf("Error running blockdev: %v", err)
 	}
 
-	partition := fmt.Sprintf("%s1", d)
+	// check if last char is numeric
+	var partition string
+	if _, err := strconv.Atoi(d[len(d)-1:]); err == nil {
```

Comment:
I think its easier to check if it is `> '0'` and `< '9'`

Reply:
fixed